CPackLists.txt

```diff
@@ -55,5 +57,5 @@ INCLUDE(CPack)
 
 
 #  2094  sudo dpkg -r g3log
-# 2100  sudo dpkg -i g3log-1.1.0-Linux.deb 
+# 2100  sudo dpkg -i g3log-1.1.0-Linux.deb
```

Comment:
Please add and change the install messages
Currently the message says
```
To install on Ubuntu		(after cmake with options && make)
make package
sudo dpkg -i g3log-***Linux.deb
```

I think it makes sense to be able to create an install package that can be transferred. 
The ```make install``` is awesome but you probably should add also the package for OSX

Please also add a printout that tells the user about the option to do 
```make install```

Please also describe what will be installed when doing ```make install```